lib/plugins/blocks.js

```diff
@@ -74,21 +75,22 @@ function inject (bot, { version }) {
 
   function delColumn (chunkX, chunkY) {
     const columnCorner = new Vec3(chunkX * 16, 0, chunkY * 16)
-    const key = columnKeyXZ(columnCorner.x, columnCorner.z)
-    delete columns[key]
+    // bot.world.unloadColumn(chunkX, chunkY) // TODO make this work
     bot.emit('chunkColumnUnload', columnCorner)
   }
 
   function addColumn (args) {
     const columnCorner = new Vec3(args.x * 16, 0, args.z * 16)
-    const key = columnKeyXZ(columnCorner.x, columnCorner.z)
     if (!args.bitMap) {
       // stop storing the chunk column
       delColumn(args.x, args.z)
       return
     }
-    let column = columns[key]
-    if (!column) columns[key] = column = new Chunk()
+    let column = bot.world.sync.getColumn(args.x, args.z)
```

Comment:
I'd add a newline to separate the former if statement from next code section

Reply:
I think we should make it work first :) but I agree we need more newline in general

Reply:
> I think we should make it work first :) 

Yup but alongside all of those (small) changes we can make code more readable!